src/idl_gen_js.cpp

```diff
@@ -220,7 +220,7 @@ class JsGenerator : public BaseGenerator {
        it != sorted_namespaces.end(); it++) {
     if (lang_.language == IDLOptions::kTs) {
       if (it->find('.') == std::string::npos) {
-        code += "import { flatbuffers } from \"./flatbuffers\"\n";
```

Comment:
hm.  I believe this actually made it work with flatbuffers as an npm module, rather than as a local file dependency.  This is otherwise irrelevant to the main changes, so take it how you like.